docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,85 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from supported formats.
+
+Selecting Data Set
```

Comment:
The Cubeviz section this links to is out of date, but I guess updating that can be a separate PR.